NppMenuSearch/Forms/FlyingSearchForm.cs

```diff
@@ -13,6 +13,13 @@ public partial class FlyingSearchForm : Form
         public FlyingSearchForm()
         {
             InitializeComponent();
+            if (Main.IsDarkModeEnabled)
+            {
+                txtSearch.BackColor = Color.FromArgb(64, 64, 64);
+                txtSearch.ForeColor = Color.FromArgb(224, 224, 224);
+                BackColor = Color.FromArgb(64, 64, 64);
+                ForeColor = Color.FromArgb(224, 224, 224);

```

Comment:
Although I do agree that hardcoded colors are generally not a good practice, please note that Windows has longstanding issues with coloring UI elements in a rational manner.  I use custom dark themes in Windows and Linux, and so many applications have unreadable text because they let Windows handle things.  (Linux handles this perfectly.)

**What I've learned is that, for Windows applications, *every* UI element needs a foreground and background color defined, and to never depend on any ***defaults*** coming from Windows.**  Depending on user-defined colors is good, but Windows unfortunately does not allow the user to define every color (and how Windows uses certain user-defined colors is ineffective, to say the least).

My thinking on how this relates to this plugin is to **get the colors from Notepad++**.  Much work has gone into the dark theming of Notepad++, and I feel it has now reached a good quality level.

(LOL... I thought I was going to write 2 sentences to quickly help out...)